.vscode/launch.json

```diff
@@ -19,6 +19,38 @@
             ],
             "internalConsoleOptions": "openOnSessionStart",
             "console": "integratedTerminal"
+        }, 
```

Comment:
This seems strange, why do we just have hooks to cli tasks rather than debugging sessions. Can we change this?

Reply:
@jamesadarich 
How do you mean? So that we can launch _something_ which allows you to try `twotime <args>` with whatever I like repeatedly?

If so, how would you achieve this?

Reply:
I assumed that you'd have for example a "debug start" and "debug finish" that will supply the args relevant for that debugging session. The other option is you instead attach and ensure that the dev start process supplies the parameters to expose node's debugging port. Does that make sense?

Reply:
As long as we attach to the process this feels better being able to use breakpoints etc :)

Reply:
We are able to set and hit breakpoints, since VSCode is launching node using the launch configuration "type" of "node".

Did you think the launch config we had here wouldn't allow setting/hitting breakpoints?

Reply:
If it works that's fine it just looks like it's pointing at the js files and I thought you had to do things like set `sourceMaps` to `true` and such for this to enable you to work with TypeScript but maybe vscode has improved this if so hooray :)

Reply:
I think that because it was compiled with source maps, it just works.